component/distributed_cache/distributed_cache.go

```diff
@@ -241,6 +242,14 @@ func (dc *DistributedCache) Stop() error {
 	fm.EndFileIOManager()
 	rm.Stop()
 	clustermanager.Stop()
+
+	// close all the RPC node client pools
+	err := rpc_client.Cleanup()
+	if err != nil {
+		log.Err("DistributedCache::Stop : Failed to cleanup RPC client pools: %v", err)
+		return err
+	}
```

Comment:
we should not check error here and not fail the Stop(). Note that we have shutdown multiple components before that, so this will leave the component in a unusable state. All various component cleanup methods must be void, unless we are ready to support undoing whatever we have sutdown.